app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -257,7 +257,7 @@ def calculate_line_43
       end
 
       def calculate_line_51
-        @intake.sales_use_tax || 0
+        @intake.sales_use_tax ? @intake.sales_use_tax.round : 0
```

Comment:
[dust] even though I know it's fine in this case, relying on truthiness of `sales_use_tax` makes me mildly uncomfortable. I have a slight preference for something like `(@intake.sales_use_tax || 0).round` instead, but I don't mind if you keep it